tutorials/site-monitoring-foundations.ipynb

```diff
@@ -0,0 +1,2737 @@
+{
```

Comment:
<p>I did not wind up using the USDA CDL in the HLS site monitoring notebook so we should update that link title.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/PlanetaryComputerExamples/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Added!